src/memory_tracker.h

```diff
@@ -1,16 +1,17 @@
-#ifndef SRC_MEMORY_TRACKER_H_
```

Comment:
use the original one would be more consistent.

Reply:
I don't agree that consistency is better then simplicity and readability.
I only changed these files since I was editing them anyway.